packages/salesforcedx-vscode-soql/package.json

```diff
@@ -134,6 +134,10 @@
         {
           "command": "soql.builder.open.new",
           "when": "sfdx:project_opened"
+        },
+        {
+          "command": "soql.builder.toggle",
+          "when": "false"
```

Comment:
@lcampos - Is there any way to write a test for a change like this?

Reply:
I think you'd have to create a system test with spectron but we've disabled those tests for the time being.

Reply:
OK. Thanks that's what i was thinking.. And i understand that is flappy.